### DISCUSSION TITLE
Badger v2 Development strategy.

### DISCUSSION THREAD

**Discussion by worldwidepixel** [worldwidepixel/badger#1] [Announcements]
Hello!

This discussion is mainly to talk about the future of Badger, with questions such as
- How should it be structured?
- How should people contribute?
- How should we handle issues and problems?
And really, anything else related to Badger. I hope that with this discussion, we can forge a clear path for Badger.

**Comment by Thinkseal** [comment]
For handling issues and problems (older that they will be patched in) they should be ordered based on how big the issue is and how hard it is to fix with 2/3 of decision be based on how big the issue is and 1/3 of it being how hard it is to fix
